### PR TITLE
add decisions document

### PR DESCRIPTION
**Pull request by komaeda**
This adds a document detailing collective-wide consensus building and decision making. It's an early writeup, and I'd welcome any and all suggestions and improvements.

/cc @johno since you were setting up the RFCs repo

### CODE REVIEW

File: decisions.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+## Local Consensus
+
+If a change is not significant enough across the entire collective to warrant an RFC, an issue may be opened on project repositories. This process follows a standard feedback loop, however the respective project pusher/merger team, as well as the organization team may close the discussion at any point if they deem necessary.
+
+In case a decision has to be made across a project or organization team, a simple Lazy Consensus method may be used. When the item has appeared to reach a consensus, the facilitator will ask "Does anyone object?" as a final call for dissent from the consensus. If an item cannot reach a consensus, a team member can call for either a closing vote or a vote to table the issue. The call for a vote must be seconded by a majority of the team or else the discussion will continue. Simple majority wins.
+
+In case consensus is split between two factions, even in the closing vote, a third-party tiebreaker may be consulted. This should be someone who has knowledge of the item at hand, and who is qualified to make a clear decision on the matter.
```
Comment (ChristianMurphy) [inline decisions.md]: Would the third party tiebreaker be the core team?

Comment (komaeda) [reply decisions.md]: I kept it intentionally vague because I'm not sure whether a core team member is always the one who can make a *clear* decision.

Comment (ChristianMurphy) [reply decisions.md]: Not _always_, but the team lead then core team should be the first options for resolution, not a third party.

Comment (wooorm) [reply decisions.md]: I think it’s good to keep it vague for now. We can revisit when we run into it!

**Pull request merged**